vdb/renderers.py

```diff
@@ -93,7 +93,7 @@ def render(self, mcanv, va):
         mcanv.addText(prefix)
 
         mcanv.addVaText(vastr, va=va)
-        mcanv.addText(": %s " % binascii.hexlify(obytes).ljust(17))
+        mcanv.addText(": %s " % binascii.hexlify(obytes).decode('utf-8').ljust(17))
```

Comment:
no changes requested here.  just a note: for a lot of things, i think we would be better off not using utf-8.  i've had times where 'utf-8' has caused problems where 'latin' has not.

Reply:
So in this case, it's doesn't really matter since we know the range of characters this should produce.

But in general I would actually argue the other way, in that especially for things that render, not just restricting ourselves to the latin character set would be better. Even if it causes apparently exceptions, utf-8 is more common than latin (at least in my experience).